broker/machinebroker/server/machine_volume_attach_test.go

```diff
@@ -89,6 +90,9 @@ var _ = Describe("AttachVolume", func() {
 			}),
 		})))
 
+		By("inspecting the effective storage resource is set for volume")
+		Expect(volume.Status.Resources.Storage().String()).Should(Equal("2k"))
+
```

Comment:
to make sure the conversion is correct in our implementation we should add some more expectations, including the binary representations (`Xi` suffixes):

- `Ki`
- `M`
- `Mi`
- `G`
- `Gi`